x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/details.ts

```diff
@@ -137,8 +137,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   }
 
   // Failing: See https://github.com/elastic/kibana/issues/129337
-  // Failing: See https://github.com/elastic/kibana/issues/129337
-  describe.skip('Rule Details', function () {
+  describe('Rule Details', function () {
```

Comment:
```suggestion
  describe.skip('Rule Details', function () {
```

only had this removed for in-pr test run.  will push it back in before we merge since we know some tests are failing and the RespOps team is looking into them soon.